tests/data/test_json.c

```diff
@@ -449,6 +449,11 @@ static const char *string_data_024 =
 "}"
 ;
 
+static const char *json_empty_ll = 
+"{"
+  "\"empty_leaf_list:ll\":[]"
```

Comment:
Still wrong, these are JSON data so prefix is the module name meaning the prefix should be `empty_leaflist_json`.